src/schema/v2/image/index.ts

```diff
@@ -70,10 +70,11 @@ export const ImageType = new GraphQLObjectType<any, ResolverContext>({
           defaultValue: 64,
         },
       },
-      resolve: ({ blurhash, aspect_ratio }, { width }) => {
-        const isBlurhashEnabled = isFeatureFlagEnabled(
-          "diamond_blurhash-enabled-globally"
-        )
+      resolve: ({ blurhash, aspect_ratio }, { width }, { userAgent }) => {
+        const isEigen = userAgent?.match("Artsy-Mobile") != null
```

Comment:
Using the same snippet from here
https://github.com/artsy/force/blob/03270b660fcf2f0bac25a16bde20b4c384b0990d/src/System/Router/buildServerAppContext.ts#L22